app/graphql/types/payment_provider_customers/provider.rb

```diff
@@ -8,6 +8,7 @@ class Provider < Types::BaseObject
       field :id, ID, null: false
       field :provider_customer_id, ID, null: true
       field :sync_with_provider, Boolean, null: true
+      field :provider_payment_methods, [String], null: false
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `provider_payment_methods` should appear before `sync_with_provider`.